exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java

```diff
@@ -776,36 +776,13 @@ private UserProfile importUserProfile(String username, boolean deleted) throws E
       LOG.trace("Remove from internal store deleted user profile '{}' from external store", username);
       organizationService.getUserProfileHandler().removeUserProfile(username, true);
     }
-
     UserProfile externalUserProfile = externalStoreService.getEntity(IDMEntityType.USER_PROFILE, username);
-
-    if (externalUserProfile != null && externalUserProfile.getUserInfoMap() != null
```

Comment:
Why do you remove the logic of comparing external and internal user profile that checks if there are modification ?